packages/round-manager/src/features/round/FundContract.tsx

```diff
@@ -349,9 +353,11 @@ export default function FundContract(props: {
           <p className="text-sm">
             {" "}
             {totalAmountLeftToFund} {matchingFundPayoutToken?.name}{" "}
-            <span className="text-sm text-slate-400 ml-2">
-              ${amountLeftToFundInUSD} USD
-            </span>
+            {amountLeftToFundInUSD > 0 ? (
```

Comment:
You should be able to use `&&` here and in other places in this PR I think?

Reply:
I didn't want to show zero value